gateway/gateway.c

```diff
@@ -67,12 +67,12 @@ int setUpGateway(int bus) {
     rfconf.tx_enable = true;
 
     if (lgw_rxrf_setconf(0, &rfconf) != LGW_HAL_SUCCESS) {
-        return EXIT_FAILURE;
+        return 2;
     }
 
     rfconf.freq_hz = RADIO_1_FREQ;
     if (lgw_rxrf_setconf(1, &rfconf) != LGW_HAL_SUCCESS) {
-        return EXIT_FAILURE;
+        return 3;
```

Comment:
Feel free to keep these if you want to build on them, or change them into your error handling style!